test/extended/deployments/deployments.go

```diff
@@ -1549,4 +1550,38 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			o.Expect(strings.TrimSpace(rcs.Items[0].Spec.Template.Spec.Containers[0].Image)).NotTo(o.BeEmpty())
 		})
 	})
+
+	g.Describe("hook inherit volume [Conformance]", func() {
+		dcName := "deployment-test"
+		g.AfterEach(func() {
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should get success logs from hook inherit inexist volume", func() {
+			dc, err := createDeploymentConfig(oc, hookInexistentVolumeFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(dc.Name).To(o.Equal(dcName))
+
+			patch := []byte(fmt.Sprintf(`[{"op": "add", "path": "/spec/strategy/rollingParams/pre/execNewPod/volumes", "value": ["v1"]}]`))
```

Comment:
I think what I mean previously was that if you burn this into the fixture like https://github.com/openshift/origin/blob/dd1f8199511a3e5b34686e5f014dc47f813d2811/test/extended/testdata/deployments/custom-deployment.yaml#L14 you don't need to repeat the test code (the Describe block) since it would be already covered there. 

Name the volume something like `non-existing-volume`